ilp-core/src/test/java/org/interledger/core/InterledgerAddressTest.java

```diff
@@ -196,6 +196,16 @@ public void testStartsWithInterledgerAddress() {
     assertThat(address.startsWith(InterledgerAddress.of("test1.foo.bob")), is(false));
   }
 
+  @Test
+  public void testStartsWithInterledgerAddressPrefix() {
```

Comment:
We should add coverage for the case where `getPrefix`is very small. For example, an address of just `g.foo`:
```
   address = InterledgerAddress.of("g.foo");
    assertThat(address.startsWith(InterledgerAddressPrefix.of("g")), is(true));
    assertThat(address.startsWith(InterledgerAddressPrefix.of("g.foo")), is(false));
    assertThat(address.startsWith(InterledgerAddressPrefix.of("g.foo.bob")), is(false));
    assertThat(address.startsWith(InterledgerAddressPrefix.of("g.foo.bar")), is(false));
    assertThat(address.startsWith(InterledgerAddressPrefix.of("test.foo")), is(false));
```